wp-content/themes/core/components/header/subheader/subheader.php

```diff
@@ -10,8 +10,25 @@
 $c = Subheader_Controller::factory( $args );
 ?>
 
-<header <?php echo $c->get_classes(); ?> <?php echo $c->get_attrs(); ?>>
-	<div class="l-container">
-		<?php get_template_part( 'components/text/text', null, $c->get_title_args() ); ?>
-	</div>
+<header <?php echo $c->get_classes(); ?> <?php echo $c->get_attrs(); ?>>		
```

Comment:
Would be nice to clean up the line spacing between elements in this file a bit (same number of empty line between sections/open-close tags).